### ISSUE TITLE
[PyTorch] Implementar a MLP

### ISSUE THREAD

**Issue by crisleymarques** [crisleymarques/postech-ml-challenge-fase-1#8]
### Objetivo
Criar a rede neural principal do projeto em PyTorch.

### Descrição
Definir a arquitetura da MLP, função de ativação, função de perda e otimizador.
Garantir que a entrada da rede esteja compatível com o pipeline de features.
Deixar a arquitetura parametrizável para experimentação.

### Checklist
- [x] Definir arquitetura da MLP
- [x] Definir função de ativação
- [x] Definir loss function
- [x] Definir optimizer
- [x] Tornar hiperparâmetros configuráveis
